front50-core/src/test/groovy/com/netflix/spinnaker/front50/model/pipeline/PipelineSpec.groovy

```diff
@@ -0,0 +1,81 @@
+package com.netflix.spinnaker.front50.model.pipeline
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.front50.api.model.pipeline.Pipeline
+import com.netflix.spinnaker.front50.api.model.pipeline.Trigger
+import com.netflix.spinnaker.front50.jackson.mixins.PipelineMixins
+import spock.lang.Specification
+
+class PipelineSpec extends Specification {
+  ObjectMapper objectMapper = new ObjectMapper()
+
+  void setup() {
+    objectMapper.addMixIn(Pipeline.class, PipelineMixins.class)
+  }
+
+  def 'should ignore correct pipeline properties from deserializing Pipeline object to JSON'() {
```

Comment:
s/from deserializing/when serializing

Probably wouldn't call it a Pipeline "object"